docker/README.md

```diff
@@ -50,6 +50,18 @@ docker-compose down
 docker-compose up
 ```
 
+Running cadence service with MySQL
+-----------------------------------------
+
+Run cadence with MySQL instead of cassandra, use following commads:
+
+```
+docker-compose -f docker-compose-mysql.yml up
+docker-compose -f docker-compose-mysql.yml down
+```
+
+Please note that SQL support is still in active developement and it is not production ready yet.
```

Comment:
thanks for calling out this !

Reply:
Does that meaning "mysql support" not production ready yet?